decidim-core/vendor/assets/javascripts/form_datepicker.js.es6

```diff
@@ -20,7 +20,7 @@
         leftArrow: '<<',
         rightArrow: '>>'
       }).on('changeDate', (ev) => {
-        $(ev.target).siblings('input').val(exports.moment.utc(ev.date));
+        $(ev.target).siblings('input').val(exports.moment(ev.date));
```

Comment:
@oriolgual This commit broke the datepicker behavior for me (I'm in Brazil). Using local time means "2017-03-25 00:00:00" will be interpreted as "2017-03-24 21:00:00", so whenever I select a date, the previous date gets sent. I discovered it because the test suite catched it:

```bash
$ cd decidim-admin; bundle exec rspec ./spec/features/admin_manages_participatory_process_steps_spec.rb[1:1:1]; cd ..
warning: parser/current is loading parser/ruby24, which recognizes
warning: 2.4.0-compliant syntax, but you are running 2.4.1.
warning: please see https://github.com/whitequark/parser#compatibility-with-ruby-mri.
Run options: include {:ids=>{"./spec/features/admin_manages_participatory_process_steps_spec.rb"=>["1:1:1"]}}
...F  HTML screenshot: file:///home/deivid/Code/decidim/spec/decidim_dummy_app/tmp/capybara/screenshot_2017-04-28-16-40-10.998.html
  Image screenshot: file:///home/deivid/Code/decidim/spec/decidim_dummy_app/tmp/capybara/screenshot_2017-04-28-16-40-10.998.png


Failures:

  1) Admin manages participatory process steps behaves like manage process steps examples creates a new participatory_process
     Failure/Error: expect(page).to have_content("12,")
       expected to find text "12," in "TITLE START DATE END DATE Eaque voluptatum magni ex quisquam voluptatem nisi recusandae. March 28, 2017 June 28, 2017 ? ? ? My participatory process step April 11, 2017 April 21, 2017 ? ? ?"
     Shared Example Group: "manage process steps examples" called from ./spec/features/admin_manages_participatory_process_steps_spec.rb:7
     # ./spec/shared/manage_process_steps_examples.rb:53:in `block (3 levels) in <top (required)>'
     # /home/deivid/.gem/ruby/2.4.1/gems/capybara-2.13.0/lib/capybara/session.rb:312:in `within'
     # /home/deivid/.gem/ruby/2.4.1/gems/capybara-2.13.0/lib/capybara/dsl.rb:52:in `block (2 levels) in <module:DSL>'
     # ./spec/shared/manage_process_steps_examples.rb:51:in `block (2 levels) in <top (required)>'
     # /home/deivid/Code/decidim/decidim-dev/lib/decidim/dev/test/rspec_support/database_cleaner.rb:10:in `block (3 levels) in <top (required)>'
     # /home/deivid/.gem/ruby/2.4.1/gems/database_cleaner-1.5.3/lib/database_cleaner/generic/base.rb:16:in `cleaning'
     # /home/deivid/.gem/ruby/2.4.1/gems/database_cleaner-1.5.3/lib/database_cleaner/base.rb:98:in `cleaning'
     # /home/deivid/.gem/ruby/2.4.1/gems/database_cleaner-1.5.3/lib/database_cleaner/configuration.rb:86:in `block (2 levels) in cleaning'
     # /home/deivid/.gem/ruby/2.4.1/gems/database_cleaner-1.5.3/lib/database_cleaner/configuration.rb:87:in `cleaning'
     # /home/deivid/Code/decidim/decidim-dev/lib/decidim/dev/test/rspec_support/database_cleaner.rb:9:in `block (2 levels) in <top (required)>'
     # /home/deivid/Code/decidim/decidim-dev/lib/decidim/dev/test/rspec_support/i18n.rb:12:in `block (2 levels) in <top (required)>'

Finished in 8.69 seconds (files took 3.93 seconds to load)
4 examples, 1 failure

Failed examples:

rspec ./spec/features/admin_manages_participatory_process_steps_spec.rb[1:1:1] # Admin manages participatory process steps behaves like manage process steps examples creates a new participatory_process
```
```bash
$ sed -i 's/exports.moment/exports.moment.utc/' decidim-core/vendor/assets/javascripts/form_datepicker.js.es6 && git diff decidim-core/vendor/assets/javascripts/form_datepicker.js.es6
diff --git a/decidim-core/vendor/assets/javascripts/form_datepicker.js.es6 b/decidim-core/vendor/assets/javascripts/form_datepicker.js.es6
index 31fa316a..3064fe3a 100644
--- a/decidim-core/vendor/assets/javascripts/form_datepicker.js.es6
+++ b/decidim-core/vendor/assets/javascripts/form_datepicker.js.es6
@@ -20,7 +20,7 @@
         leftArrow: '<<',
         rightArrow: '>>'
       }).on('changeDate', (ev) => {
-        $(ev.target).siblings('input').val(exports.moment(ev.date));
+        $(ev.target).siblings('input').val(exports.moment.utc(ev.date));
       });
     });
   };
```

```bash
$ cd spec/decidim_dummy_app; bundle exec rake assets:precompile; cd ../..
$ cd decidim-admin; bundle exec rspec ./spec/features/admin_manages_participatory_process_steps_spec.rb[1:1:1]; cd ..
warning: parser/current is loading parser/ruby24, which recognizes
warning: 2.4.0-compliant syntax, but you are running 2.4.1.
warning: please see https://github.com/whitequark/parser#compatibility-with-ruby-mri.
Run options: include {:ids=>{"./spec/features/admin_manages_participatory_process_steps_spec.rb"=>["1:1:1"]}}
....

Finished in 6.25 seconds (files took 3.93 seconds to load)
4 examples, 0 failures
```

What was the rationale behind the change?